packages/protocol-http/src/httpHandlerInitialization.browser.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Alternate compilation for browser excluding http Agent types.
+ *
+ * @internal
+ */
+export type DefaultRequestHandlerInitializationPassThroughParameters = FetchHttpHandlerOptions;
```

Comment:
should these types and interfaces live in `@smithy/types`, and the `@smithy/*-http-handler` packages import them so there is no duplication?